fusesoc/main.py

```diff
@@ -298,6 +301,127 @@ def core_sign(fs, args):
     print(f"{sigfile} created")
 
 
+def guess_provider():
+    guess = {"found": False}
+    cmd = ["git", "remote", "-v"]
+    res = subprocess.run(cmd, capture_output=True).stdout.decode("utf-8").strip()
```

Comment:
Let's run this from the root directory of the core instead. We can pick up the directory from `core.core_root` in `core_publish` and send it to `guess_provider` as an argument. The reason is that most FuseSoC users have a workspace which is separate from the directories where the cores are.